default.nix

```diff
@@ -1,6 +1,12 @@
-haskellPackages: rec {
+haskellPackages: nixpkgs: rec {
+  which = haskellPackages.callHackageDirect {
+    pkg = "which";
+    ver = "0.1.0.0";
+    sha256 = "1c8svdiv378ps63lwn3aw7rv5wamlpmzgcn21r2pap4sx7p08892";
+  } {};
+
   gargoyle = haskellPackages.callCabal2nix "gargoyle" ./gargoyle {};
   gargoyle-postgresql = haskellPackages.callCabal2nix "gargoyle-postgresql" ./gargoyle-postgresql { };
-  gargoyle-postgresql-nix = haskellPackages.callPackage ./gargoyle-postgresql-nix { };
+  gargoyle-postgresql-nix = import ./gargoyle-postgresql-nix haskellPackages nixpkgs;
```

Comment:
This can be `haskellPackages.callPackage ./gargoyle-postgresql-nix { inherit nixpkgs; }`

However, that's longer and your way actually makes it move visually obvious that this line is different from the rest, which has tricked me before.

Reply:
```nix
{
  # ...
  gargoyle-postgresql-nix = haskellPackages.callPackage ({ pkgs, postgresql }:
    pkgs.haskell.lib.overrideCabal (haskellPackages.callCabal2nix "gargoyle-postgresql-nix" ./gargoyle-postgresql-nix { }) (drv: {
      librarySystemDepends = (drv.librarySystemDepends or []) ++ [ postgresql ];
    }));
}
```

Reply:
What exactly are you proposing? To inline the override for system depends? I thought it would be better not to inline that here so that if you only plan to override the `gargoyle-postgresql-nix` you won't get screwed if you pull in the directory itself instead of this wrapper.